build/docker/Dockerfile-bare

```diff
@@ -0,0 +1,43 @@
+# This dockerfile builds the zap stable release
+FROM ubuntu:16.04 as builder
+MAINTAINER Simon Bennetts "psiinon@gmail.com"
+
+WORKDIR /zap
+
+RUN apt-get update && apt-get install -q -y --fix-missing \
+	wget \
+	curl \
+	xmlstarlet \
+	unzip && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/*
+
+# Download and expand the latest stable release 
+RUN curl -s https://raw.githubusercontent.com/zaproxy/zap-admin/master/ZapVersions-dev.xml | xmlstarlet sel -t -v //url |grep -i Linux | wget --content-disposition -i - -O - | tar zxv && \
+	cp -R ZAP*/* . &&  \
+	rm -R ZAP* 
+
+FROM java:openjdk-8-jdk-alpine
+MAINTAINER Simon Bennetts "psiinon@gmail.com"
+
+WORKDIR /zap
+COPY --from=builder /zap .
+RUN echo "http://dl-3.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories &&\
+    apk add --update --no-cache bash && \
+    adduser -h /home/zap -s /bin/bash zap -D zap && \
+    rm -rf /var/cache/apk/* && \
+    chown zap /zap && \
+	chgrp zap /zap && \
+    chown -R zap:zap /zap
+
+#Change to the zap user so things get done as the right person (apart from copy)
+USER zap
+
+ENV PATH $JAVA_HOME/bin:/zap/:$PATH
+ENV ZAP_PATH /zap/zap.sh
+
+# Default port for use with zapcli
```

Comment:
Those env variables are needed, I'll remove the port variable. Also noticed that I need to update the `healthcheck` as it depends on `zap-cli`.